src/virt.c

```diff
@@ -1458,8 +1465,17 @@ static int lv_connect(void) {
     int status = virNodeGetInfo(conn, &nodeinfo);
     if (status != 0) {
       ERROR(PLUGIN_NAME " plugin: virNodeGetInfo failed");
+      virConnectClose(conn);
```

Comment:
is this done to allow reconnect attempts?

Reply:
I added this for integrity. If `lv_connect()`  fails on any its stage, then no other callbacks will try to run on partially initialized plugin.

Reply:
ok, works for me.